tests/v0_tests/test_bulk_search.py

```diff
@@ -667,7 +666,92 @@ def test_query_validation_two_same_urls_success(self):
         if marqo_cloud_url_before_test:
             os.environ["MARQO_CLOUD_URL"] = marqo_cloud_url_before_test
 
-    @mock_instance_mappings([InstanceMappingIndexData("test-index", "NOT READY", "example.com")])
+    @mark.ignore_during_cloud_tests
+    def test_opensource_can_bulk_search_2_indices(self):
+        self.client.create_index(self.generic_test_index_name)
+        self.client.create_index(self.generic_test_index_name_2)
+
+        d1 = {
+            "doc title": "Very heavy, dense metallic lead.",
+            "abc-123": "some text blah",
+            "multiply": 2,
+            "add": 1,
+            "_id": "my-cool-doc"
+        }
+        d2 = {
+            "doc title": "The captain bravely lead her followers into battle."
+                         " She directed her soldiers to and fro.",
+            "field X": "this is a solid doc blah",
+            "field1": "other things",
+            "multiply": 2,
+            "add": 1,
+            "_id": "123456"
+        }
+
+        self.client.index(self.generic_test_index_name).add_documents([d1], tensor_fields=["doc title"])
+        self.client.index(self.generic_test_index_name_2).add_documents([d2], tensor_fields=["doc title"])
+
+        for search_method in [enums.SearchMethods.TENSOR]:
```

Comment:
can we do `in [enums.SearchMethods.TENSOR, enums.SearchMethods.LEXICAL]:` while we're here?